straxen/plugins/afterpulse_processing.py

```diff
@@ -87,21 +82,21 @@ def compute(self, raw_records):
 
         # calculate baseline and baseline rms
         strax.baseline(records,
-                       baseline_samples=self.config['baseline_samples'],
+                       baseline_samples=self.baseline_samples,
```

Comment:
This doesn't hurt, but perhaps - if we start doing a full sweep of strax-options, using the `self.config` reference is not an issue? Just to keep the review readable

Reply:
The issue with leaving the `self.config` code is that it can cause confusion if someone assumes the config can be set to a URL but then gets errors because the plugin is reading the URL directly instead of accessing it through the getattr interface where the URL is interpreted.

Reply:
okay, thanks for reminding me. Probably this shouldn't be impossible to fix on the plugin side. But okay, maybe we shouldn't over-complicate and indeed just use the `getattr` interface

Reply:
Right, in theory we could make a subclass of `immutabledict` that does all the magic when you try to access one of its keys and use it as the config dict for plugins. I actually thought about using this approach instead of descriptors when i first made URLConfig but ended up going with descriptors because i wanted to leave the ability to access the raw config value from the plugin in case someone needs to do some hack